### ISSUE TITLE
Warning: Undefined index: serviceKeyword

### ISSUE THREAD

**Issue by m-vo** [contao/easy-coding-standard#7]
This warning gets reported when running the cs-fixer:

```
PHP Notice:  Undefined index: serviceKeywords in […]/vendor/symplify/easy-coding-standard/src/Yaml/CheckerServiceParametersShifter.php on line 72
```

Issue: https://github.com/symplify/symplify/issues/2873
Fix: https://github.com/symplify/symplify/pull/2874

The fix was only released in the current major version (9) of the bundle. I couldn't find anything about their release policy, but it seems there is only ever support for the newest version. So maybe we would need to migrate to 9 as well…?

**Comment by leofeyer** [comment]
Fixed in 8fffe9dcba4d17a95b1bd1645ba62679efcae6e3.
